src/app/pages/details/common/publication-info.js

```diff
@@ -236,25 +213,25 @@ export default function PublicationInfo({model, url, polish}) {
     return (
         <React.Fragment>
             <LabeledDate
-                label={texts.pub}
+                label={pub}
                 className="loc-pub-date"
                 date={model.publishDate}
             />
             <LabeledDate
-                label={texts.web}
+                label={web}
                 className="loc-web-update-date"
                 date={webUpdate}
             />
             <PdfUpdateInfo updateDate={model.lastUpdatedPdf} url={url} />
-            <IsbnInfo model={model} label={texts.hard} tag="print" />
-            <IsbnInfo model={model} label={texts.paper} tag="printSoftcover" />
-            <IsbnInfo model={model} label={texts.dig} tag="digital" />
+            <IsbnInfo model={model} label={hard} tag="print" />
+            <IsbnInfo model={model} label={paper} tag="printSoftcover" />
+            <IsbnInfo model={model} label={dig} tag="digital" />
             <IsbnInfo
                 model={model}
-                label={model.ibookVolume2Isbn10 || model.ibookVolume2isbn13 ? `iBooks ${texts.part} 1` : 'iBooks'}
+                label={model.ibookVolume2Isbn10 || model.ibookVolume2isbn13 ? `iBooks ${part} 1` : 'iBooks'}
                 tag="ibook"
```

Comment:
This is probably fine for English & Spanish, but things like this get really tricky in some languages.  For instance the verb might come last or the number be formatted differently.  Looks like format-js will support that